modules/dnn/src/torch/THDiskFile.cpp

```diff
@@ -3,6 +3,7 @@
 #include "THGeneral.h"
 #include "THDiskFile.h"
 #include "THFilePrivate.h"
+#include <string>
```

Comment:
I believe it should be located in `.hpp` file, because `std::string` is part of external interface.